### PR TITLE
Select again

### PR DESCRIPTION
**Pull request by feenster**

### CODE REVIEW

File: client/app/project/project.controller.js [hunk 1]
```diff
@@ -521,6 +521,93 @@
             });
         }
 
+        vm.hasTaskLockedForMapping = function () {
```
Comment (LindaAlblas) [inline client/app/project/project.controller.js]: Add some docs

File: client/app/project/project.controller.js [hunk 2]
```diff
@@ -521,6 +521,93 @@
...
+            if (vm.taskVectorLayer) {
+                var lockedFeatures = taskService.getTaskFeaturesByIdAndStatus(vm.taskVectorLayer.getSource().getFeatures(), vm.lockedTasksForCurrentUser, 'LOCKED_FOR_MAPPING');
+                return lockedFeatures.length == 1;
+            }
+
+        };
+
+        vm.reselectTaskForMapping = function () {
```
Comment (LindaAlblas) [inline client/app/project/project.controller.js]: Add some docs

File: client/app/project/project.controller.js [hunk 3]
```diff
@@ -521,6 +521,93 @@
...
+                    selectFeature(lockedFeatures[0]);
+                    onTaskSelection(lockedFeatures[0]);
+                    var padding = getPaddingSize();
+                    vm.map.getView().fit(lockedFeatures[0].getGeometry().getExtent(), {padding: [padding, padding, padding, padding]});
+                }
+            }
+
+        };
+
+        vm.hasTasksLockedForValidation = function () {
```
Comment (LindaAlblas) [inline client/app/project/project.controller.js]: Add some docs

File: client/app/project/project.controller.js [hunk 4]
```diff
@@ -521,6 +521,93 @@
...
+            if (vm.taskVectorLayer) {
+                var lockedFeatures = taskService.getTaskFeaturesByIdAndStatus(vm.taskVectorLayer.getSource().getFeatures(), vm.lockedTasksForCurrentUser, 'LOCKED_FOR_VALIDATION');
+                return lockedFeatures.length > 0;
+            }
+        };
+
+        vm.reselectTasksForValidation = function () {
```
Comment (LindaAlblas) [inline client/app/project/project.controller.js]: Add docs

File: client/app/project/project.controller.js [hunk 5]
```diff
@@ -521,6 +521,93 @@
...
+                        vm.resetTaskData();
+                        vm.currentTab = 'validation';
+                        vm.validatingStep = 'multi-locked';
+                        vm.multiSelectedTasksData = tasksLockedForValidation;
+                        vm.multiLockedTasks = tasksLockedForValidation;
+                        vm.isSelectedValidatable = true;
+
+                    }, function (error) {
+                        // TODO - handle error
+                        // onLockError(vm.projectData.projectId, error)
```
Comment (LindaAlblas) [inline client/app/project/project.controller.js]: remove commented out line?

**Comment by feenster** [comment]
Fixes #617

**Pull request merged**